transfer-hbar/script-transfer-hbar.js

```diff
@@ -115,7 +115,7 @@ async function scriptTransferHbar() {
     const transferJsonAccountTransfersFinalAmounts = transferJsonAccountTransfers
         ?.map((obj) => Hbar.from(obj.amount, HbarUnit.Tinybar).toString(HbarUnit.Hbar));
     logger.log(
-        'The debit and credit amounts of the transfer transaction:\n',
+        'The debit, credit, and transaction fee amounts of the transfer transaction:\n',
```

Comment:
This change is replicated in #9
See https://github.com/hedera-dev/hello-future-world-js/pull/10/files#r1701160360

Reply:
once this is merged the second PR will automatically update with not modifying this no?